src/pages/Frontpage.jsx

```diff
@@ -120,18 +121,19 @@ function Frontpage(props) {
           })}
         />
         <br></br>
-        <br></br>
 
 
-        <label>Właściciel</label>
+        <label>Właściciel </label>
         <input
           {...register("owner", { required: true })}
           placeholder="Wpisz imię właściciela"
         />
-        {errors.owner && <div className="text-red-500 text-sm">Właściciel jest wymagany</div>}
-        <button className="main primary" href={`${props.id}`}>
-          Potwierdź
-        </button>
+        {errors.owner && <div className="text-red-500 text-sm">Właściciel jest wymagany </div>}
+        <br></br>
+        <br></br>
+        <Button
+      className="p-3 transition ease-in-out duration-500 font-semibold rounded-xl"
```

Comment:
p-3 transition ease-in-out duration-500 font-semibold rounded-xl

to by można było trzymać już w samym buttonie

clue jest takie że jak będziesz w innym miejscu buttona używał to powinieneś jak najmniej rzeczy powtarzać a te style odpowiadają za animację przycisku więc w sumie to można je do tego cn() wrzucić na stałe